### ISSUE TITLE
Any Regression example?

### ISSUE THREAD

**Issue by yogeshhk** [graph4ai/graph4nlp#351]
## ❓ Questions and Help

Is there any example, with supervised learning learning dataset having, sentences and corresponding scores (float)?

train.csv:
```
When the young people returned they changed appearance.                 -0.340259125
And outside before the palace a great garden was walled round.           -1.054013390
```

If there is nothing as ready example, which steps do you suggest to accomplish this?

**Comment by AlanSwift** [comment]
Thank you for your consistent interest in our project!
It seems that you propose a regression problem. However, I'm sorry that there is no direct solution in our examples. But I can give you my insights and I hope that they can help you.
You can borrow the idea from text classification (the last softmax layer can be removed to fit your output and use MSE loss). Concretely, you can employ graph construction methods such as dependency graphs and then encode with GNNs such as GCN. Then you can project it to the numeric number that you want. The loss function can be MSE or others.

**Comment by AlanSwift** [comment]
I will close this issue.
